.github/workflows/build.yml

```diff
@@ -437,6 +437,11 @@ jobs:
           cat >> _update_spec << EOF
           # This file is used for regulating self-update
           lock 2022.08.18.36 .+ Python 3.6
+          lock 2023.11.14 .+ Python 3.7
+          lockV2 yt-dlp/yt-dlp 2022.08.18.36 .+ Python 3.6
+          lockV2 yt-dlp/yt-dlp 2023.11.14 .+ Python 3.7
+          lockV2 yt-dlp/yt-dlp-nightly-builds 2023.11.14.232721 .+ Python 3.7
+          lockV2 yt-dlp/yt-dlp-master-builds 2023.11.14.214307 .+ Python 3.7
           EOF
```

Comment:
```suggestion
          lock 2022.08.18.36 .+ Python 3\.6
          lock 2023.11.14 .+ Python 3\.7
          lockV2 yt-dlp/yt-dlp 2022.08.18.36 .+ Python 3\.6
          lockV2 yt-dlp/yt-dlp 2023.11.14 .+ Python 3\.7
          lockV2 yt-dlp/yt-dlp-nightly-builds 2023.11.14.232721 .+ Python 3\.7
          lockV2 yt-dlp/yt-dlp-master-builds 2023.11.14.214307 .+ Python 3\.7
          EOF
```
It's regex

Reply:
it's only regex after the repository. it's one of hte things we had to change in V2 (or else all channels would match the pattern where the channel is wildcarded)